src/view/components/formikReactSpectrum3/radioGroupWithDataElement.jsx

```diff
@@ -12,11 +12,7 @@ governing permissions and limitations under the License.
 
```

Comment:
Following the naming conventions so far this should probably be named "src/view/components/formikReactSpectrum3/formikRadioGroupWithDataElement.jsx"

Reply:
Ah, yes. This wasn't intentional. Thanks for bringing it up.